synfig-core/src/synfig/CMakeLists.txt

```diff
@@ -134,11 +137,11 @@ target_link_libraries(synfig
         ${GLIBMM_LINK_LIBRARIES}
         ${GIOMM_LINK_LIBRARIES}
         ${LIBXML_LINK_LIBRARIES}
-        ${XMLPP_LIBRARIES}
+        ${XMLPP_LINK_LIBRARIES}
```

Comment:
We are trying to support CMake 3.5 for Linux old stable versions.
Can you check if `<XPREFIX>_LDFLAGS` is work here?